frontend/src/metabase/writeback/components/ActionCreator/FormCreator/utils.ts

```diff
@@ -65,26 +68,32 @@ const fieldPropsTypeMap: FieldPropTypeMap = {
   datetime: "date",
   monthyear: "date",
   quarteryear: "date",
+  email: "email",
+  password: "password",
+  number: "integer", // this input type is badly named, it works for floats too
+  boolean: "boolean",
+  category: "category",
   dropdown: "select",
-  "inline-select": "radio",
+  radio: "radio",
 };
 
 const inputTypeHasOptions = (fieldSettings: FieldSettings) =>
-  ["dropdown", "inline-select"].includes(fieldSettings.inputType);
-
-interface FieldProps {
-  type: string;
-  options?: OptionType[];
-  values?: any;
-  placeholder?: string;
-  validate?: Validator;
-}
+  ["dropdown", "radio"].includes(fieldSettings.inputType);
 
-const getParameterFieldProps = (fieldSettings: FieldSettings) => {
-  const fieldProps: FieldProps = {
+const dontValidate = () => undefined;
```

Comment:
nit: we can use [`_.noop` (doc)](https://underscorejs.org/#noop) in cases like that